src/company_gui.cpp

```diff
@@ -53,6 +53,7 @@
 /** Company GUI constants. */
 static void DoSelectCompanyManagerFace(Window *parent);
 static void ShowCompanyInfrastructure(CompanyID company);
+bool _infrastructure_window_open = false;
```

Comment:
Based on the other globals used in viewport.cpp, maybe a better name for this would be `_viewport_highlight_company_infrastructure` ?